Akkadu/rsi-api-widgets
# Chore/lerna2

Please describe the purpose of the pull request.

* the purpose to this pull request was to implement lerna in this repo.

## Motivation and Context
* Updating manually the new packages and dependencies was quite painful and messy...
* Lerna is now solving this issue

## How To Test
* Please follow the steps described in the `README-akkadu-dev.md`,[link](https://github.com/Akkadu/rsi-api-widgets/blob/chore/lerna2/README-akkadu-dev.md)
  *  you should be able to install the project and run all the packages with `yarn start` (without getting any errors...)

## Screenshots (if appropriate)

## Checklist



加油！
